registry/tests/test.spec.ts

```diff
@@ -3,6 +3,6 @@ import { request, expect } from './common';
 describe('Tests /', () => {
     it('Hello world', async () => {
         const response = await request.get('/');
-        expect(response.text).to.be.equal('Hello! This is Micro Fragments registry service.');
+        expect(response.text).to.contain(' <title>React Admin</title>');
```

Comment:
I guess, space is not necessary here (' <title...')

Reply:
updated